authorisationtest/jwt_tokens.go

```diff
@@ -3,6 +3,7 @@ package authorisationtest
 // The following JWT tokens have been generated for use in tests.
 // They can be verified using the default public key provided in the configuration.
 
+//nolint:gosec // tokens intended for testing, so they do not constitue any security issue
```

Comment:
(suggestion): probably worth adding this```authtest``` package into the golangcilint.yml so that these linters won't be run against these test files ? But yeah this is also fine I guess.

Reply:
maybe we should keep it, as this package may be used by other services?